types/flatry/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What's this file here for?

Reply:
I got errors without this file:
<img width="1010" alt="Screenshot 2022-10-03 at 08 25 54" src="https://user-images.githubusercontent.com/26653756/193501763-f37a172d-b56b-4565-a68d-c5cd01f332eb.png">

Reply:
This file definitely shouldn't be here, you may want to delete `node_modules` and reinstall as your copy of dtslint was likely out of date.